index.html

```diff
@@ -635,14 +635,6 @@ <h2>
           act as follows:
         </p>
         <ol data-link-for="PaymentDetailsBase" class="algorithm">
-          <li data-tests=
-          "allowpaymentrequest/active-document-cross-origin.https.sub.html, allowpaymentrequest/active-document-same-origin.https.html, allowpaymentrequest/removing-allowpaymentrequest.https.sub.html, allowpaymentrequest/setting-allowpaymentrequest-timing.https.sub.html, allowpaymentrequest/setting-allowpaymentrequest.https.sub.html">
-          If the <a>current settings object</a>'s <a data-cite=
```

Comment:
This is redundant now, given:
https://github.com/w3c/webappsec-feature-policy/blob/master/features.md#payment

@domenic, agree?

Reply:
No, this is wrong. The source of truth for how the PaymentRequest constructor behaves needs to be this spec. The fact that https://github.com/w3c/webappsec-feature-policy/blob/master/features.md#payment has a MUST requirement is just confusing. That document should be updated: <del>MUST throw a SecurityError</del><ins>will throw in the manner described in the PaymentRequest specification</ins>.

/cc @clelland to consider changes to avoid such confusion.

Reply:
For "V1" we are trying to avoid putting a normative dependency on Feature Policy (bc: document remains unpublished, unclear if WebKit will support it ATM). 

What I'd like to do instead is just say: "If the current settings object's responsible document is not allowed to use the API, throw a SecurityError" - implying that Feature Policy would deem the API usage "not allowed"... that hopefully gives us enough of a hook into Feature Policy without having a normative dependency on it. 

I can then point the reader to the sections Feature Policy and `allowpaymentrequest` below.

Reply:
Original text was actually fine, because it points to `allowpaymentrequest`, which itself names the "payment" feature and points to Feature Policy (while meeting our current requirements). Reverting.